libs/wire-api/src/Wire/API/Routes/Named.hs

```diff
@@ -42,17 +42,22 @@ newtype Named name x = Named {unnamed :: x}
 -- types other than string literals in some places.
 class RenderableSymbol a where
   renderSymbol :: Text
+  renderOperationId :: Text
+  renderOperationId = renderSymbol @a
 
 instance (KnownSymbol a) => RenderableSymbol a where
   renderSymbol = T.pack . show $ symbolVal (Proxy @a)
+  renderOperationId = T.pack $ symbolVal (Proxy @a)
```

Comment:
is `renderSymbol` still used now?  if not you could just change its behavior instead of adding another method to the class.